.github/workflows/build.yml

```
@@ -0,0 +1,67 @@
+name: CI
+
+on: [push]
+
+jobs:
+  build:
+    name: Build and Test on node ${{ matrix.node_version }} and ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        version: [14, 16]
+        os: [ubuntu-latest, windows-latest, macOS-latest]
```

```suggestion
        os: [ubuntu-latest]
```

Right now there shouldn't be any need for running this on windows or macOS